File: templates/python/bayes_uq_before_after.py
"""bayes_uq_before_after: 贝叶斯与不确定性量化前后斜率对比（slope 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='Bayesian uncertainty quantification: before-after slope'):
    return make_template_figure('slope', seed=4320, title=title, domain='Bayesian uncertainty quantification', topic='before-after slope')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/bayes_uq_before_after.m
function fig = bayes_uq_before_after()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('slope', 4320, 'Bayesian uncertainty quantification: before-after slope', 'Bayesian uncertainty quantification', 'before-after slope');
end
